.github/workflows/multi_service_image_build.yml

```diff
@@ -54,32 +58,53 @@ jobs:
           do 
             if [[ -e $DIR/src && -e $DIR/Dockerfile ]]
             then
+              DIR=`echo $DIR| sed -r 's/\/$//'`
               CHANGED_WLS_DIRS+=("$DIR")
             fi  
           done
           echo "CHANGED_WLS_DIRS=${CHANGED_WLS_DIRS[@]}">> "$GITHUB_ENV"
-          for wl_service in ${CHANGED_WLS_DIRS[@]}
+          for wl_service_dir in ${CHANGED_WLS_DIRS[@]}
           do
+              wl_service=`basename ${wl_service_dir}`
+              if [[ ${wl_service_dir} =~ ^app ]]
+                then 
+                    DOCKERFILE="${wl_service_dir}/Dockerfile"
+                    BUILD_CONTEXT=""
+                    echo "APP_STRUCT=1">>"$GITHUB_ENV"
+                else
+                    DOCKERFILE="Dockerfile"
+                    BUILD_CONTEXT="${wl_service_dir}"
+              fi
               argo submit .argo/wl-service-wf.yaml \
                     --generate-name ${wl_service}-wf- \
-                    -p repo=git@github.com:<GIT_ORGANIZATION_NAME>/<WL_REPO_NAME>.git \
+                    -p repo=git@github.com:cgdevx-demo/vt-cnask.git \
```

Comment:
hardcoded values, need replace to parameters, as for me, need to replace git@github.com with some new parameter to have support for GitLab in the future.

Reply:
Fixed, thank you.